### PR TITLE
Add Docker cluster

### PR DESCRIPTION
**Pull request by adriendupuis**
Configure a cluster of Docker containers atop eZ Platform EE 2.5.9
- Varnish
- Apache w/ PHP-FPM & eZ
- Redis
- MariaDB
- Solr

Please, see [docker/README.md](https://github.com/adriendupuis/ezplatform-ee-docker/blob/feat_container_cluster/docker/README.md) for more.

### CODE REVIEW

File: docker/apache/parameters.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    #env(SESSION_HANDLER_ID): redis
+    #env(SESSION_SAVE_PATH): tcp://redis:6379
+
+    env(DATABASE_HOST): mariadb
+    env(DATABASE_PASSWORD): root
+    env(DATABASE_VERSION): mariadb-MARIADB_VERSION
+
+    #TODO: Fix "can not sort on multivalued field: path_string_id"
+    #env(SEARCH_ENGINE): solr
+    #env(SOLR_DSN): http://solr:8983/solr
```
Comment (adriendupuis) [inline docker/apache/parameters.yml]: With Solr 6.6.6, eZ Platform EE 2.5.9 (ezsystems/ezpublish-kernel 7.5.7) and ezsystems/ezplatform-solr-search-engine 1.7.3:
- I got a `can not sort on multivalued field: path_string_id` Solr error on /admin home;
- I got a `An exception has been thrown during the rendering of a template ("Notice: Array to string conversion").` on /admin/dashboard.

Comment (andrerom) [reply docker/apache/parameters.yml]: Hmm, how does this differ from docker setup in ez platform repo on solr setup?

Comment (adriendupuis) [reply docker/apache/parameters.yml]: @andrerom Thank you. By looking at doc/docker/Dockerfile-solr I immediately saw what is missing: I forgot to load the vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/schema.xml so multivalued was used as a default for every fields :disappointed:

Comment (adriendupuis) [reply docker/apache/parameters.yml]: Fixed in https://github.com/adriendupuis/ezplatform-ee-docker/pull/1/commits/756b10a4ebebc099201ea3972aac8d0b2a68dc02

File: docker-compose.bash [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+while [ -n "`echo $MARIADB_VERSION | grep 'ERROR';`" ]; do
+  echo 'Waiting for server inside mariadb container...';
+  sleep 3;
+  MARIADB_VERSION=`eval $GET_MARIADB_VERSION_CMD`;
+done;
+echo "MariaDB version: $MARIADB_VERSION";
+
+# Apache: Symfony parameters.yml
+cp docker/apache/parameters.yml app/config/parameters.yml;
+sed -i '' -e "s/MARIADB_VERSION/$MARIADB_VERSION/" app/config/parameters.yml;
```
Comment (adriendupuis) [inline docker-compose.bash]: The `-i ''` argument particularity is due to Mac OS X's sed. To be less specific and more generic, a solution could be https://formulae.brew.sh/formula/gnu-sed

Comment (adriendupuis) [reply docker-compose.bash]: Added to TODO list in e6dc638c2745e6c491addf91c2f660ddbf606436

**Pull request merged**